tasks/main.yml

```diff
@@ -22,6 +22,16 @@
   include_tasks: ssl.yml
   when:  proto == "https"
 
+- name: Enable /info page acl
+  ansible.builtin.lineinfile:
+    path: "{{ nginx_include_path }}/{{ nginx_docservice_file }}"
+    insertafter: 'location ~* ^(\/[\d]+\.[\d]+\.[\d]+[\.|-][\d]+)?\/(info)(\/.*)$ {'
+    search_string: 'allow'
+    line: '  allow {{ ds_info_allow }};'
+  when: ds_info_allow
+  notify:
+      restart-nginx
```

Comment:
It is necessary to use a capital letter here, because we have updated the role structure in this pr #46, and now all task names starting [from capital letter](https://github.com/ONLYOFFICE/ansible-role-documentserver/pull/46/files#diff-67da321934d1a76cffaa1feed7ef7899327b68724357089bf6eeb4af62103715R10)

Should be: 
```suggeestion
   Restart-nginx
```